src/components/Enumerate.tsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react'
 import { connect } from 'react-redux';
 import { startDeviceScan, stopDeviceScan } from '../store/device';
-import { AppState } from '../store';
+import {AppState, Loadable} from '../store';
 import { Spinner } from 'reactstrap';
+import {Device, Wallet} from "../types";
 
 interface StateProps {
-  devices: AppState['device']['devices'];
+  devices: Loadable<Device[]>;
```

Comment:
IMO I like `AppState` being the single source of truth here. If we update the reducer's types at all, those updates propagate to all of the components. In this case, if we changed it from `Loadable<Device[]>` to something else, we would get an error down in mapStateToProps, as opposed to where the usage would actually change in the component.

Reply:
This syntax makes more sense to me after reading up on the indexed access operator.